src/test/java/com/wikia/webdriver/common/core/elemnt/Wait.java

```diff
@@ -223,7 +189,7 @@ public boolean forTextInElement(List<WebElement> elements, int index, String tex
     try {
       elements.get(0).getTagName();
     } catch (WebDriverException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)